### ISSUE TITLE
Does y_pred in the loss function need to be activated? Is it okay to keep it as logits?

### ISSUE THREAD

**Issue by yesulove** [jocpae/clDice#32]

**Comment by aryaabdi** [comment]
Good question! I've been wondering the same thing.
